src/shared/components/navbar/pageMap.tsx

```diff
@@ -9,46 +9,55 @@ const pageMap: {
     permission: null,
     label: 'dashboard.label',
     path: '/',
+    icon: 'dashboard',
   },
   newPatient: {
     permission: Permissions.WritePatients,
     label: 'patients.newPatient',
     path: '/patients/new',
+    icon: 'patient-add',
   },
   viewPatients: {
     permission: Permissions.ReadPatients,
     label: 'patients.patientsList',
     path: '/patients',
+    icon: 'incident',
   },
   newAppointment: {
     permission: Permissions.WriteAppointments,
     label: 'scheduling.appointments.new',
     path: '/appointments/new',
+    icon: 'appointment-add',
   },
   viewAppointments: {
     permission: Permissions.ReadAppointments,
     label: 'scheduling.appointments.schedule',
     path: '/appointments',
+    icon: 'incident',
   },
   newLab: {
     permission: Permissions.RequestLab,
     label: 'labs.requests.new',
     path: '/labs/new',
+    icon: 'add',
   },
   viewLabs: {
     permission: Permissions.ViewLabs,
     label: 'labs.requests.label',
     path: '/labs',
+    icon: 'incident',
   },
   newIncident: {
     permission: Permissions.ReportIncident,
     label: 'incidents.reports.new',
     path: '/incidents/new',
+    icon: 'add',
   },
   viewIncidents: {
     permission: Permissions.ViewIncidents,
     label: 'incidents.reports.label',
     path: '/incidents',
+    icon: 'incident',
   },
   settings: {
     permission: null,
```

Comment:
we have a settings icon that we can use here.

Reply:
Instead of `incident` for `viewIncidents`(the marked line) or for the `settings` field? I assume the settings field?

Do you know the name of the icon?

Reply:
For the settings field, sorry the marked line got a little weird. 

The name of the icon is called `setting`. 

For future reference, here are a list of our icon set: https://components.hospitalrun.io/?path=/story/icons--icon-set